diagrams-svg.cabal

```diff
@@ -61,6 +62,7 @@ Library
                      , lens                 >= 4.0   && < 4.12
                      , hashable             >= 1.1   && < 1.3
                      , optparse-applicative >= 0.10  && < 0.12
+                     , semigroups
```

Comment:
Our policy is to always put in upper bounds.  You are right that in most cases the upper bound is not necessary.  But you have to think about who suffers when things break.  With upper bounds, we have to test and upload a new version every time a new version of a package comes out, but ideally our users never notice.  With no upper bound, our users experience the breakage when they try to install diagrams and it uses a new version of something that doesn't work.